docs/working_with_data_sources.md

```diff
@@ -0,0 +1,222 @@
+---
+id: working_with_data_sources
+author: GoodData
+sidebar_label: Working with Data Sources
+title: Working with Data Sources
+---
+
+Creating a Data Source
+-------
+
+You can create a Snowflake data source programmatically:
```

Comment:
You can create a Snowflake Data Source programmatically:

Reply:
Done fix it